diff --git MODULE.bazel MODULE.bazel
index 20de49e..68f783c 100644
--- MODULE.bazel
+++ MODULE.bazel
@@ -59,10 +59,6 @@ single_version_override(
 
 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
 go_deps.from_file(go_mod = "//:go.mod")
-go_deps.gazelle_override(
-    build_file_generation = "on",
-    path = "github.com/cncf/xds/go",
-)
 use_repo(
     go_deps,
     "cc_mvdan_gofumpt",
